runtime/common/src/constants.rs

```diff
@@ -85,13 +85,51 @@ pub type FrameSystemMaxConsumers = ConstU32<16>;
 // --- MSA Pallet ---
 /// The maximum number of public keys per MSA
 pub type MsaMaxPublicKeysPerMsa = ConstU8<25>;
-/// The maximum size of the provider name (in bytes)
-pub type MsaMaxProviderNameSize = ConstU32<16>;
 /// The number of blocks per virtual bucket
 pub type MSAMortalityWindowSize = ConstU32<{ 20 * MINUTES }>;
 /// The upper limit on total stored signatures.
 /// Set to an average of 50 signatures per block
 pub type MSAMaxSignaturesStored = ConstU32<50_000>;
+/// The maximum size of a provider name (in bytes)
```

Comment:
Any reason these can't just be, say `ConstU32<256>`, for example?

Reply:
ConstU32 does not implement some additional macros needed by pallet/extrinsic constraints, so need a custom type to satisfy those bounds

Reply:
Bummer.